sdk/nodejs/restAPI.ts

```diff
@@ -148,4 +149,9 @@ export interface RestAPIArgs {
      * Either `swaggerString` or `routes` must be specified.
      */
     swaggerString?: pulumi.Input<string>;
+    /**
+     * 'Map of tags to assign to the resource. If configured with a provider `default_tags` configuration block present,
```

Comment:
Prefer to inflect defaultTags? There is no one good answer here since default_tags is how it works in Python, but defaultTags is how it works in Node, but Node is more commonly used.

Reply:
I actually copied this from the docs of the underlying API gateway resource.

I wanted to stay in line with what we have on AWS classic, but I'm open to changing it if you think that'll be clearer

Reply:
Slight preference for defaultTags but no blocker.

Reply:
Changed it!